packages/tasit-action/src/TasitAction.test.js

```diff
@@ -131,7 +158,35 @@ describe("Contract", function() {
     await mineBlocks(simpleStorage.getProvider(), 8);
   });
 
-  it.skip("should send a signed message", async () => {});
+  it("should throw error when subscribing on invalid event", async () => {
+    const events = ["ValueChanged", "InvalidEvent"];
+    try {
+      const subscription = simpleStorage.subscribe(events);
+      assert(false, "subscription with invalid event");
+    } catch (e) {
+      assert(true);
+    }
+  });
 
-  it.skip("should listen to an event", async () => {});
+  it("should throw error then listening on invalid event", async () => {
```

Comment:
This test has the same name as the previous one

Reply:
One is `subscribing` and the other is `listening`.

Reply:
Ah ok, my mistake

Reply:
This looks good then